src/saml2/metadata.py

```diff
@@ -770,6 +766,13 @@ def entity_descriptor(confd):
         entd.authn_authority_descriptor = do_aq_descriptor(confd, mycert,
                                                            enc_cert)
 
+    conf_node_country = confd.getattr('node_country', confd.context)
+    if conf_node_country:
+        if not entd.extensions:
+            entd.extensions = md.Extensions()
+        item = node_country.NodeCountry(text=conf_node_country)
```

Comment:
Are there specific values that `node_country` should have? Do we need to validate the value?

Reply:
Specific values meaning default ones? No, but we do need to validate them because they have to be in ISO 3166-1 alpha-2 format (the xsd defines it as [A-Z][A-Z]). I don't believe this is the right point for the validation though